docs/001_develop/03_client-capabilities/008_interaction/number-field.mdx

```diff
@@ -0,0 +1,357 @@
+
+    Declaration
+    ```html
+    <rapid-number-field></rapid-number-field>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-text-field :value=${sync((x) => x.fieldValue)}>
+        </rapid-text-field>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      @observable fieldValue = '';
+      fieldValueChanged() {
+        console.log(this.fieldValue);
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+    <rapid-number-field></rapid-number-field>
+    ```
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const handleChange = (event) => {
+        console.log(event.target.value)
+      };
+      return (
+        <rapid-number-field onChange={handleChange}></rapid-number-field>
+      );
+    }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+    <rapid-number-field></rapid-number-field>
+    ```
+
+    Usage
+    ```typescript
+    import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+    import { FormsModule } from '@angular/forms';
+    @Component({
+      selector: 'my-root',
+      template: `
+        <rapid-number-field
+          (change)="onChange($event)"
+        ></rapid-number-field>
+      `,
+      styleUrls: ['./my.component.css'],
+      standalone: true,
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [FormsModule],
+    })
+    export class AppComponent {
+      onChange(event: Event) {
+        const target = event.target as HTMLInputElement;
+        console.log(target.value);
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax.
+
+### Attributes
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+      <th>Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>autofocus</td>
+      <td><code>boolean</code></td>
+      <td>When true, the component will be focused when the page has finished loading</td>
+<td>
+
+```typescript
+<rapid-number-field ?autofocus="${(x) => x.autoFocus}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>appearance</td>
+      <td><code>string</code></td>
+      <td>Can be <code>outline</code> or <code>filled</code></td>
+<td>
+
+```typescript
+<rapid-number-field appearance="outline">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>disabled</td>
+      <td><code>boolean</code></td>
+      <td>Similar to <code>readonly</code>, but with a blur on the component</td>
+<td>
+
+```typescript
+<rapid-number-field ?disabled="${(x) => x.isDisabled}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>locale</td>
+      <td><code>string</code></td>
+      <td>Defines a number format based on language and location. **Default: "en-US"**. Must be used with `withFormatting`</td>
+<td>
+
+```typescript
+<rapid-number-field locale="en-GB">
```

Comment:
As this should be used with `withFormatting` it could be good to add that attribute into this example too